packages/stacked_themes/lib/src/theme_manager.dart

```diff
@@ -196,10 +196,17 @@ You can supply either a list of ThemeData objects to the themes property or a li
   }
 
   /// Swaps between the light and dark ThemeMode
-  void toggleDarkLightTheme() {
+  ///
+  /// If [disablePersistence] is set to true, the selected theme will
+  /// not persist across app restarts.
+  void toggleDarkLightTheme([bool disablePersistence = false]) {
     _selectedThemeMode =
         _selectedThemeMode == ThemeMode.dark ? ThemeMode.light : ThemeMode.dark;
 
+    if (disablePersistence == false) {

```

Comment:
Should this be added to this PR?

Reply:
Yes please, if that would be easier to review, so you can close the other PR and the relevant issue(s).

Reply:
I just find it out of scope. doesn't relate to the PR title in any way.

Reply:
Alright, I understand, you can remove it, thanks.

Reply:
What do you want me to remove the whole PR?

Reply:
I was thinking to drop that particular commit actually, so you would only deal with the code relevant to this PR